autoarray/util/nn/nn_py.py

```diff
@@ -105,4 +105,4 @@ def natural_interpolation(x_in, y_in, z_in, x_target, y_target):
 
 except OSError:
     print("natural neighbour interpolation not loaded.")
-    pass
+    raise ImportError
```

Comment:
Maybe raise with the explanation rather than printing then raising?

```python
raise ImportError("natural neighbour interpolation not loaded.")
```

Although a more descriptive message would be better